Example/FinancialConnections Example/FinancialConnections Example/Playground/PlaygroundConfiguration.swift

```diff
@@ -0,0 +1,475 @@
+                return email
+            } else {
+                return ""
+            }
+        }
+        set {
+            configurationStore[Self.emailKey] = newValue
+        }
+    }
+
+    // MARK: - Permissions
+
+    private static let balancesPermissionKey = "balances_permission"
+    var balancesPermission: Bool {
+        get {
+            if let balancesPermission = configurationStore[Self.balancesPermissionKey] as? Bool {
+                return balancesPermission
+            } else {
+                return false
+            }
+        }
+        set {
+            configurationStore[Self.balancesPermissionKey] = newValue
+        }
+    }
+    private static let ownershipPermissionKey = "ownership_permission"
+    var ownershipPermission: Bool {
+        get {
+            if let ownershipPermission = configurationStore[Self.ownershipPermissionKey] as? Bool {
+                return ownershipPermission
+            } else {
+                return false
+            }
+        }
+        set {
+            configurationStore[Self.ownershipPermissionKey] = newValue
+        }
+    }
+    private static let paymentMethodPermissionKey = "payment_method_permission"
+    var paymentMethodPermission: Bool {
+        get {
+            if let paymentMethodPermission = configurationStore[Self.paymentMethodPermissionKey] as? Bool {
+                return paymentMethodPermission
+            } else {
+                return false
+            }
+        }
+        set {
+            configurationStore[Self.paymentMethodPermissionKey] = newValue
+        }
+    }
+    private static let transactionsPermissionKey = "transactions_permission"
+    var transactionsPermission: Bool {
+        get {
+            if let transactionsPermission = configurationStore[Self.transactionsPermissionKey] as? Bool {
+                return transactionsPermission
+            } else {
+                return false
+            }
+        }
+        set {
+            configurationStore[Self.transactionsPermissionKey] = newValue
+        }
+    }
+
+    // MARK: - Other Settings
+
+    private static let liveEventsKey = "live_events"
+    var liveEvents: Bool {
+        get {
+            if let liveEvents = configurationStore[Self.liveEventsKey] as? Bool {
+                return liveEvents
+            } else {
+                return false
+            }
+        }
+        set {
+            configurationStore[Self.liveEventsKey] = newValue
+        }
+    }
+
+    // MARK: - Update
+
+    func updateConfigurationString(_ configurationString: String) {
+        guard
+            let jsonData = configurationString.data(using: .utf8),
+            let jsonObject = try? JSONSerialization.jsonObject(with: jsonData, options: []),
+            let dictionary = jsonObject as? [String: Any]
+        else {
+            // this prevents anyone from overriding the configuration with something wrong
+            return
```

Comment:
is it worth asserting here or anything so they can try to make sense of it?

Reply:
yea you are spot on that this would help - right now descoping as this should be rare (90%+ of the use of this code will not interact with this in a way that this `return` will fire)